docs/providers/documentation/grafana-provider.mdx

```diff
@@ -69,3 +69,9 @@ You can check that the Grafana Provider works by testing Keep's contact point (w
   <img height="10" src="/images/grafana_sa_5.png" />
 </Frame>
 5. Go to Keep - you should see an alert from Grafana!
+
+## Webhook Integration Modifications
+
+The webhook integration adds Keep as a contact point in the Grafana instance. Keep also gains access to the following scopes:
+- `alert.provisioning:read`
+- `alert.provisioning:write`
```

Comment:
@talboren is this the correct format ? If so, I shall proceed to add this side note to every provider that supports the webhook integration.

Reply:
Yep! It's worth mentioning how to find this contact point and what its name is. By the way, I see there are more scopes here https://docs.keephq.dev/providers/documentation/grafana-provider#connecting-with-the-provider. Do we ask for unnecessary scopes by mistake?

Reply:
> Yep! It's worth mentioning how to find this contact point and what its name is. By the way, I see there are more scopes here https://docs.keephq.dev/providers/documentation/grafana-provider#connecting-with-the-provider. Do we ask for unnecessary scopes by mistake?

I think it's just the screenshot I took before clicking on Apply ^^'